project/plugins.sbt

```diff
@@ -1,9 +1,11 @@
-addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.15")
+val scalaJSVersion = Option(System.getenv("SCALAJS_VERSION")).getOrElse("0.6.19")
```

Comment:
Why are you pulling the version from the environment ? Can't we just set it to `1.0.0-M2` or something?

Reply:
That's the recommended way to cross-build across Scala.js 0.6.x and 1.x: https://www.scala-js.org/news/2017/11/29/announcing-scalajs-1.0.0-M2/